awx_collection/tests/integration/targets/tower_user/tasks/main.yml

```diff
@@ -13,6 +13,17 @@
     that:
       - "result is changed"
 
+- name: Check idempotency
```

Comment:
I cannot overstate how happy it makes me that we actually have a meaningful approach to testing this now.